axelrod/tests/strategies/test_player.py

```diff
@@ -260,7 +260,7 @@ def attribute_equality_test(self, player, clone):
                 self.assertTrue(np.array_equal(reset_value, original_value),
                                 msg=attribute)
 
-            elif isinstance(reset_value, types.GeneratorType):
+            elif isinstance(reset_value, types.GeneratorType) or isinstance(reset_value, itertools.cycle):
```

Comment:
> how do i reference your previous comment about this?

Sorry, I'm not sure what you mean. I think this is fine as it is. No need to reference anything.